tests/unit/javascript/test_greasemonkey.py

```diff
@@ -26,6 +26,9 @@
 from PyQt5.QtCore import QUrl
 
 from qutebrowser.browser import greasemonkey
+# pylint: disable=unused-import
+from tests.unit.browser.test_adblock import download_stub
+# pylint: enable=unused-import
```

Comment:
Importing fixtures from other test files can lead to funny side-effects. Instead, you should move `FakeDownloadItem` and `FakeDownloadManager` to `tests/helpers/stubs.py` and the fixture to `tests/helpers/fixtures.py`.